### ISSUE TITLE
VOXCONVERT:  OBJ VOXELIZE TO VOX，Does not carry colors

### ISSUE THREAD

**Issue by Finchmc** [vengi-voxel/vengi#449]
The work you do is very interesting and I can't wait to start trying it out, but I'm having a bit of a problem. Looking forward to your answers! 
I refer to this statement from https://vengi-voxel.github.io/vengi/voxconvert/Examples/#voxelize-a-obj-gltf-ply-fbx-stl-or-any-supported-mesh-file   ：
Voxelize a obj, gltf, ply, fbx, stl or any supported mesh file ：
_**./vengi-voxconvert -set voxformat_scale 2 -set palette /path/to/palette.png -set voxformat_createpalette false --input mesh.obj --output voxels.vox**_
Here are my local commands (win)：
_**vengi-voxconvert.exe -set voxformat_scale 3 -set palette D:/project/3d_color/bbb.png -set voxformat_createpalette false --input D:/project/3d_color/baby_dragon.obj --output D:/project/3d_color/baby_dragon_2.vox**_

But I can't get the vox file with the colours as per the help manual's instructions.
The first image is the effect I wanted to get
![baby_dragon](https://github.com/vengi-voxel/vengi/assets/166574443/97f412bc-b924-4508-aee7-5de62cc1444c)
, while the second image is the all-black vox file generated by typing the above command.
![no_color_dragon](https://github.com/vengi-voxel/vengi/assets/166574443/56a86aa0-46c9-4196-938b-25bf407fd04f)

By the way, when I want to use a ply file to voxelize a vox file. 
![robot](https://github.com/vengi-voxel/vengi/assets/166574443/0627c865-b569-4577-9ad0-13e2d6ecb5d1)

vengi-voxconvert -set voxformat_scale 200 --input D:/project/3d_color/robot.ply --output D:/project/3d_color/robot1.vox
Failed to load model file D:/project/3d_color/robot.ply Scene graph doesn't contain models.

Thank you again for your excellent work and I look forward to answering your queries, much appreciated!

**Comment by mgerhardy** [comment]
Thanks you very much for your report.

I have a few questions: 

* which version of vengi are you using? (`vengi-voxconvert.exe --version` )
* Does the `obj` file has the `mtl` file with the texture properly referenced in? (`map_Kd` in the `mtl` file)
* Is it possible to attach the `ply` and `obj/mtl/png` files to this issue as a **zip** file? (looks like the ply has some special face configuration and I need to fix something here)

**Comment by Finchmc** [comment]
- Thank you very much for taking the time to reply to my report. According to your reminder, I have successfully obtained vox through the following instructions:

**_./vengi-voxconvert -set voxformat_scale 2 -set palette /path/to/palette.png -set voxformat_createpalette false --input mesh.obj --output voxels.vox_**

- But the effect of the vox graph I got is not good. I think it may be related to my improper operation. I sincerely ask you, can I generate a vox file with the following characteristics through an obj file? The generated vox file has bright color levels and a high degree of pure color. Generally speaking, black is black, white is white, and red is red. While the color boundaries are clear, the overall VOX file has few color categories.

![图片1](https://github.com/vengi-voxel/vengi/assets/166574443/4cc6d0e4-3a7e-4f50-b2a6-6749b60f0b82)

- The left image is the vox image obtained by other algorithms, the middle is the obj file, and the right image is the vox image obtained by your algorithm (vengi-voxconvert.exe). I would like to ask if there is any relevant parameter adjustment in your code that can realize the above idea.

- Finally I sent you the zip package of the file for your reference

[house_fine.zip](https://github.com/vengi-voxel/vengi/files/14940142/house_fine.zip)
Thank you again for your excellent work and I look forward to answering your queries, much appreciated!!!
